src/dropdown/dropdown.scss

```diff
@@ -24,7 +24,9 @@ $menu-background: $brc-white !default;
 $menu-border-color: $brc-silver !default;
 $option-color: rgba($brc-darkGrey, .5) !default;
 $option-color-hover: $brc-darkGrey !default;
+$option-color-selected: $option-color !default;
 $option-background-hover: rgba($brc-azure, .15) !default;
+$option-background-selected: $menu-background !default;
```

Comment:
to avoid making this issue "breaking" I set `option-color-selected` and `option-background-selected` to the defaults (this is what we were getting before v1.0.0-rc1 and is what we still expect in QIA)